local_planner_node:
  show_config: true
  debug: true
  
  #ws_x_max: 6
  #ws_y_max: 6
  #ws_x_min: 0
  #ws_y_min: 0

  #map_resolution: 0.05
  
  goal_weight: 1.2
  cost_weight: 0.25
  lof_distance: 1.5
  occ_threshold: 99

  world_frame: map
  robot_base_frame: base_link

